app/Resources/translations/forms.fr.xlf

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="utf-8"?>
+<xliff xmlns="urn:oasis:names:tc:xliff:document:1.2" xmlns:jms="urn:jms:translation" version="1.2">
+  <file source-language="en" target-language="en" datatype="plaintext" original="not.available">
+    <header>
+      <tool tool-id="JMSTranslationBundle" tool-name="JMSTranslationBundle" tool-version="1.1.0-DEV"/>
+      <note>The source node in most cases contains the sample message as written by the developer. If it looks like a dot-delimitted string such as "form.label.firstname", then the developer has not provided a default message.</note>
+    </header>
+    <body>
+      <trans-unit id="50c33e7e9f7fff89c34710fb134161c1e4fd0dae" resname="role.policy.personalization">
+        <source>Personalization</source>
+        <target>Personalization</target>
+        <note>key: role.policy.content</note>
+      </trans-unit>
+      <trans-unit id="107ce4d94dfeb4b9e77d83c539d2db22184589b8" resname="role.policy.personalization.all_functions">
+        <source>Personalization / All functions</source>
+        <target>Personalization / All functions</target>
+        <note>key: role.policy.personalization.all_functions</note>
+      </trans-unit>
+      <trans-unit id="edafc6d69ed57b5971c704c6092e404d460c86d3" resname="role.policy.personalization.show_menu_item">
+        <source>Personalization / Show menu item</source>
+        <target>Personalization / Show menu item</target>
```

Comment:
As commented below, my recommendation is to use `Personalization / View`.
So in French it will be `Personnalisation / Voir`